src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+const ADD = 'ADD_BOOK';
+const REMOVE = 'REMOVE_BOOK';
+
+const booksXer = (state = [{ id: '1', title: 'Elden Ring', author: 'Miyazaki' }, { id: '2', title: 'One Piece', author: 'Eichiro Oda' }], action) => {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state,
+        action.book,
+      ];
+    case REMOVE:
```

Comment:
- [ ] Remember to update your action types here if you changed it from the requested change above.